content/en/blog/2025/how-to-name-your-spans.md

```diff
@@ -0,0 +1,131 @@
+---
+title: How to Name Your Spans
+linkTitle: How to Name Your Spans
+date: 2025-08-06
```

Comment:
```suggestion
date: 2025-08-08
```

Let's have this either tomorrow or Monday since we published the OTiP blog post today